gce_ovf_import_tests/test_suites/ovf_import_test_suite.go

```diff
@@ -71,10 +78,27 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	suffix := utils.RandString(5)
 	logger.Printf("Running TestSuite %q", testSuite.Name)
 
-	startupScript, err := ioutil.ReadFile("gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	startupScriptUbuntu3disks, err := ioutil.ReadFile(
+		"gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	if err != nil {
+		os.Exit(1)
```

Comment:
Just an exit is a bit abrupt and hard to diagnose, at least log the error

Reply:
Added logging and refactored script loading to a separate func